### PR TITLE
Add Extraction request properties

### PR DESCRIPTION
**Pull request by rkkr**

### CODE REVIEW

**Comment by MindaugasButkus** [comment]
Add changelog entry.

File: src/test/ExtractionClient.test.ts [hunk 1]
```diff
@@ -55,6 +55,36 @@ describe("ExtractionClient", () => {
...
+    const extraction = await extractionClient.runExtraction("auth", "iModelId", extractionRequest);
+    expect(extraction.id).to.be.eq(1);
+    expect(fetchStub.calledWith(
+      "https://api.bentley.com/insights/reporting/datasources/imodels/iModelId/extraction/run",
+      "pass",
+    )).to.be.true;
+    expect(requestStub.calledWith(
+      "POST",
+      "auth",
+      JSON.stringify(extractionRequest),
```
Comment (MindaugasButkus) [inline src/test/ExtractionClient.test.ts:86]: That's just a copy paste of the implementation. I would rather get the argument from a call and assert each of the objects properties individually.

Comment (rkkr) [reply src/test/ExtractionClient.test.ts:86]: `createRequest` is called with just 3 strings.

Comment (MindaugasButkus) [reply src/test/ExtractionClient.test.ts:86]: `JSON.parse(requestStub.getCall(...).args[2])` and it is no longer a string.

Comment (rkkr) [reply src/test/ExtractionClient.test.ts:86]: I don't see point in doing this. No other test is parsing request bodies. And if we want to not rely on `JSON.stringify` we can just expect a preserialized string.

**Pull request merged**